### PR TITLE
[CI] change 'dir-commit' variable.

### PR DESCRIPTION
**Pull request by ohsewon**
swap 'input_pr' and 'input_date'. It makes easily find repo folder and change path. in example, type {pr-number} and push tab.

Fixed issue #410.

Signed-off-by: sewon.oh <sewon.oh@samsung.com>

---

### CODE REVIEW

File: ci/taos/checker-pr-audit-async.sh [hunk 1]
```diff
@@ -103,7 +103,7 @@ export dir_worker=$dir_worker
 # check if dir_commit folder exists, then, create dir_commit folder
 # let's keep the existing result although the same target directory already exists.
 cd $dir_ci
-export dir_commit=${dir_worker}/${input_date}-${input_pr}-${input_commit}
+export dir_commit=${dir_worker}/${input_pr}-${input_date}-${input_commit}
```
Comment (leemgs) [inline ci/taos/checker-pr-audit-async.sh]: Which better?
1) display the folders with prefix "pr_date" to monitor the number of PRs per a day. (Currently)
2) display the folders with prefix "pr_no" to find repo folders of a PR easily.

**Comment by leemgs** [comment]
* After this PR
   * Two links will be used  for  developers ("Field") and administrator("Last modified") as following:
image

File: ci/taos/checker-pr-audit-async.sh (hunk 1 above)
Comment (leemgs) [reply ci/taos/checker-pr-audit-async.sh]: @ohsewon  Please append the below statement in the commit body (=commit description) for history. 

`Fixed issue #410.`

Comment (ohsewon) [reply ci/taos/checker-pr-audit-async.sh]: updated.

**Comment by leemgs** [comment]
* Since 21:04 Nov-14-2018

image

**Pull request merged**